src/diem/testing/suites/conftest.py

```diff
@@ -94,10 +94,22 @@ def travel_rule_threshold(diem_client: jsonrpc.Client) -> int:
 
 
 @pytest.fixture
-def pending_income_account(stub_client: RestClient) -> AccountResource:
+def stub_wallet_pending_income_account(stub_client: RestClient) -> AccountResource:
+    """MiniWallet stub saves the payment without account information (subaddress / reference id)
+    into a pending income account before process it.
+    """
+
     return AccountResource(id=PENDING_INBOUND_ACCOUNT_ID, client=stub_client)
 
 
+@pytest.fixture(autouse=True)
+def log_stub_wallet_pending_income_account(
+    stub_wallet_pending_income_account: AccountResource,
+) -> Generator[None, None, None]:
+    yield
```

Comment:
why do you need this?

Reply:
simplified answer:
1. before yield code is executed before test method run
2. after yield code is executed after test method run

see https://docs.pytest.org/en/stable/fixture.html#yield-fixtures-recommended

here we need print log after test is done

Reply:
ah ok thank you!